ConsoleApp/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace ConsoleApp
+{
+    using System;
+    class ConsoleApp
+    {
+        static void SelectionSort(int[] arr)
+        {
+            int founMinValue;
+            for (int i = 0, j = 0; i < arr.Length; i++)
+            {
+                founMinValue = arr[i];
+                int minIndex = 0;
+                int replaceElement = arr[i];
+                for (j = i+1; j < arr.Length; j++)
+                {
+                    if(arr[j] < founMinValue)
+                    {
+                        minIndex = j;
+                        founMinValue = arr[j];
+                        arr[i] = founMinValue;
+                    }
+                }
+                if (minIndex != 0)
+                {
+                    arr[minIndex] = replaceElement;
+                }
+            }
+        }
+        static void printArray(int [] arr, string comment)
```

Comment:
please use correct name convention, it should be named **PrintArray**

Reply:
updated